### PR TITLE
[wip] Expand list of potential incubation sponsors to all global owners

### PR DESCRIPTION
**Pull request by mattymo**

### CODE REVIEW

**Comment by mattymo** [comment]
The link in the PR points to people who have any OWNER rights. I am still trying to find the correct link that shows all k8s org-wise owners.

**Comment by deads2k** [comment]
> The link in the PR points to people who have any OWNER rights. I am still trying to find the correct link that shows all k8s org-wise owners.

You're probably looking for https://github.com/orgs/kubernetes/people?utf8=%E2%9C%93&query=%20role%3Aowner , but I don't see a strong reason to change the list sponsors.  I think the original logic stands:

> The idea is that by relying on this small set of Kubernetes Community members to make a determination on Incubator projects we will ensure that there is consistency around new projects joining the Incubator.

File: incubator.md [hunk 1]
```diff
@@ -36,7 +36,7 @@ The next discussion should be on a relevant Special Interest Group mailing list.
 
 The final process is to email kubernetes-dev@googlegroups.com to announce your intention to form a new Incubator project. Include your entire proposal in the body of the email and prefix the Subject with [Incubator]. Include links to your discussion on the accepted SIG mailing list to guide the discussion.
 
-Acceptance of the project into the Kubernetes Incubator happens once a Sponsor approves. Anyone listed as an approver in the top-level pkg OWNERS file https://github.com/kubernetes/kubernetes/blob/master/pkg/OWNERS can sponsor a project by replying to the kubernetes-dev discussion with LGTM.
+Acceptance of the project into the Kubernetes Incubator happens once a Sponsor approves. Anyone listed as an approver in the top-level pkg OWNERS file https://github.com/kubernetes/kubernetes/blob/master/pkg/OWNERS or [Kubernetes project owners](https://github.com/kubernetes/kubernetes/people) can sponsor a project by replying to the kubernetes-dev discussion with LGTM.
```
Comment (derekwaynecarr) [inline incubator.md]: the link appears to be incorrect.

**Comment by idvoretskyi** [comment]
CC @philips @bgrant0607 @sarahnovotny as the initial authors of "incubator proposal".

**Comment by bgrant0607** [comment]
cc @kubernetes/contributor-experience

**Comment by bgrant0607** [comment]
@mattymo What are you trying to achieve and what is the rationale? It's not helpful that there's no PR description.

Clearly we need PR and issue templates for this repo, and perhaps extend the bot to close PRs and issues that don't have any descriptions.

**Comment by mattymo** [comment]
I think it's already decided that expanding the list of sponsors to include github-wide Kubernetes owners is not the preferred solution. I'll just close this.

**Pull request closed without merging**